packages/api/src/routes/medical/document.ts

```diff
@@ -277,6 +273,7 @@ async function getUploadUrlAndCreateDocRef(req: Request): Promise<UploadDocument
 router.post(
   "/upload-url",
   requestLogger,
+  patientAuthorization("query"),
```

Comment:
we're we never validating this?

Reply:
must happen in `getUploadUrlAndCreateDocRef(req);` -- likely we should remove it there and use `const { cxId, id: patientId } = getPatientInfoOrFail(req);` here?